manifests/kube-state-metrics-deployment.yaml

```diff
@@ -31,6 +31,13 @@ spec:
         - --telemetry-port=8082
         image: quay.io/coreos/kube-state-metrics:v1.9.7
         name: kube-state-metrics
+        resources:
```

Comment:
These differ from upstream recommendations, IIRC you did some measurements yourself?

Reply:
Yes, look at https://github.com/prometheus-operator/kube-prometheus/pull/868#discussion_r553375880

Reply:
It would be good to include any recommended resource requirements/limits in the ksm mixins though. Right now there's nothing here: https://github.com/kubernetes/kube-state-metrics/blob/master/jsonnet/kube-state-metrics/kube-state-metrics.libsonnet#L164-L180